### PR TITLE
DRIVERS-1596 Remove usages of xtrace in shell scripts

### PR DESCRIPTION
**Pull request by alcaeus**
DRIVERS-1596

The `xtrace` shell option prints all commands that are executed, which may reveal sensitive information. It also clutters the build logs, making debugging issues a lot harder.

While it's useful to use `xtrace` to ensure commands and options are assembled correctly, this should only be used while developing and not as part of the regular usage.

### CODE REVIEW

File: .evergreen/config.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ functions:
            PROJECT_DIRECTORY: "$PROJECT_DIRECTORY"
            PREPARE_SHELL: |
               set -o errexit
-              set -o xtrace
+              set +o xtrace
```
Comment (divjotarora) [inline .evergreen/config.yml:72]: Why the change to `+o xtrace` here rather than removing the existing line as was done in other files?

Comment (alcaeus) [reply .evergreen/config.yml]: I had originally left this in to disable xtrace when preparing a shell, but any new shell most likely starts out with it disabled anyways. I've removed it 👍

**Review by divjotarora — APPROVED** [review]
